modules/ROOT/pages/connected-apps-org-admin.adoc

```diff
@@ -2,37 +2,84 @@
 
-You can use the *Authorizations* section to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details or to remove the app from your organization.
-When you revoke an app’s access, the app’s OAuth 2.0 access tokens and refresh tokens are revoked, and the app can no longer access user data.
+=== Edit the App Owner
 
-You can configure one of two types of authorization policies for your users:
+Organization administrators can edit the owner of the connected app and transfer the ownership to other administrators:
 
-* Unrestricted
+. In Access Management > *Connected Apps* > *Owned Apps*, click the app name. 
+. Make the changes and click *Save changes*.
 
-  Organization members can authorize any connected app to act on their behalf.
+[[modify-client-secret]]
+=== Modify the Client Secret for a Connected App
 
-* Restricted
+include::partial$include-nav-connected-apps.adoc[]
+
+. Click a connected app from the list to enable Edit mode.
+. In the *Secret* field, enter a secret of your choice.
+. Click *Save*.
+
+image::connected-apps-edit-mode.png[Edit mode for a connected app]
++
+[calloutlist]
+. The name and username of the owner of the connected app
+. The name of the connected app
+. The ID of the connected app
+. The client secret of the connected app
+
+=== View Authorizations
+
+. In the *Authorizations* tab, select the app.
+. Click the more actions menu (*...*) for the application and select *Show details*.
 
-  Only connected apps that are added to an allowlist by an administrator can be authorized by organization members to act on their behalf.
+=== Revoke Authorization 
+
+. In the *Authorizations* tab, select the app.
+. Click the more actions menu (*...*) for the selected application and select *Remove*.
+. Click *Remove*. 
+
+When you revoke an app’s access, the app’s OAuth 2.0 access tokens and refresh tokens are revoked, and the app can no longer access user data. This action can't be undone.
+
+== Configure Authorization Policies
+
+You can configure one of two types of authorization policies for your users:
+
+* Unrestricted
++
+Organization members can authorize any connected app to act on their behalf.
+* Restricted
++
+Only connected apps that are added to an allowlist by an administrator are authorized by organization members to act on their behalf.
 
 By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application allowlist is enabled, and all existing application authorizations are added automatically.
-When using the **Restricted** policy, you may disable auto-approval of all internal apps, in which case each internal app must be added to the allowlist like any other connected app.
+When using the **Restricted** policy, you can disable auto-approval of all internal apps, in which case you must add each internal app to the <<application-allowlist-admin,application allowlist>> like any other connected app.
```

Comment:
Double ** OK?

Reply:
Split into two sentences, I think.

Reply:
The ** is markdown and renders correctly. This was prob written by a dev originally.